README.md

```diff
@@ -328,3 +328,32 @@ Now, for any new html page, we need to do the following:
 {% endblock %}
 
 ```
+
+## Generate ERD with django extensions
+```
+ pip install graphviz
+ pip install django-extensions
+ ```
+ Go to settings.py and add 
+ ```
+ 'django_extensions',
+ ```
+to INSTALLED_APPS
+
```

Comment:
Go to `settings.py` file and the following line to `INSTALLED_APPS`:
 ```
 'django_extensions',
 ```
For better readability

Reply:
Got it